src/lib/chips/chips.scss

```diff
@@ -82,6 +85,7 @@ $mat-chips-chip-margin: 8px;
 .mat-chip-remove {
   margin-right: $mat-chip-remove-margin-after;
   margin-left: $mat-chip-remove-margin-before;
+  cursor: pointer;
```

Comment:
I had previously removed it because of this animation https://material.io/guidelines/components/chips.html#chips-behavior, but I certainly understand how pointer helps show purpose